0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,272 @@
+[canceled](#canceled) or [executed](#executed).
+
+## Forward action
+For a sender, the forward action consists in creating an outgoing transfer.
+When a connector is notified that an incoming transfer has reached the 'prepared' state, its forward action is to propose a transfer on the next ledger,
+with the same hashlock condition, and the same ILP packet attached.
+For a receiver, there is no forward action.
+
+## Hop
+One connector-facilitated connection between one ledger and the next.
+The number of hops is always one less than the number of transfers, so a one-transfer ("local") transaction is a zero-hop transaction.
+
+## IL-RFC
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in
+[gh:interledger/rfcs](https://github.com/interledgers/rfcs).
+
+## Incoming (transfer, ledger, amount)
+Relative to a connector, the transfer/ledger/amount that is located inbetween the connector and the sender, directly adjacent to the connector.
+For a receiver, the adjacent transfer/ledger/amount is always incoming.
+A sender does not have an incoming transfer/ledger/amount.
+
+## Interledger (in "Let's use Interledger for that!")
+The [Interledger protocol stack](#interledger-protocol-stack).
+
+## interledger (in "An interledger")
+Any network of two or more ledgers connected via the Interledger protocol stack. See also [Interledger (The)](#interledger-the)
+
+## Interledger (in "The Interledger")
+The main public network of ledgers connected via the Interledger protocol stack. See [ConnectorLand](https://connector.land) for statistics about it.
+
+## Interledger Addresses
+Interledger addresses provide a ledger-agnostic way to address ledgers and accounts. Interledger addresses are dot-separated strings that contain prefixes to group ledgers. An example address might look like `g.us.acmebank.acmecorp.sales.199` or `g.crypto.bitcoin.1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2`.
+When initiating an Interledger payment, the sender attaches an ILP packet to the local transfer to the connector. The packet is a binary message that includes the destination account, destination amount, and additional data for the receiver. The packet is relayed by connectors and attached to each transfer that comprises the payment. In some cases, ledger protocols may define alternative ways to communicate the packet.
+Note that anyone can claim to have a certain Interledger address, there is no registry of them.
+Whether or not a transaction ends up at the intended receiver is ultimately safe-guarded by the hashlock condition,
+not by enforcement of address ownership.
+See [IL-RFC 15](../0015-ilp-addresses/0015-ilp-addresses.md)
+
+## Interledger Architecture
+The [Interledger protocol stack](interledger-protocol-stack), and the design principles behind it.
+
+## Interledger layer
+The lowest layer of the Interledger protocol stack, consisting of
```

Comment:
There's an interesting open question about whether the ledger layer is part of the "Interledger protocol stack"